go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22.0
 toolchain go1.22.1
 
 require (
+	github.com/adhocore/gronx v1.19.0
```

Comment:
is this the same parser used by k8s?

Reply:
no, it is the cron scheduler I found while searching, this was on the initial PR